run-tests.sh

```diff
@@ -12,7 +12,7 @@
 # or submit itself to any jurisdiction.
 
 pydocstyle idutils tests && \
-isort -rc -c -df **/*.py && \
+isort -c -df **/*.py && \
```

Comment:
Actually `-df` is now `--diff` (https://timothycrosley.github.io/isort/docs/upgrade_guides/5.0.0/). Let's expand the options anyway since shorthand is only useful to skip on typing and since this is a script we don't need to skimp on that ;) 

 ```suggestion
isort --check-only --diff **/*.py && \
```